### PR TITLE
json schema validation extensions

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

**Comment by fenollp** [comment]
Is it possible to not have to add extended-validation? My rationale is for the json schemas to contain all the required info, that way newcomers/users can get up to speed faster just reading from the "familiar" json files.

File: applications/crossbar/priv/couchdb/schemas/callflows.action.json [hunk 1]
```diff
@@ -13,11 +13,14 @@
             "type": "object"
         },
         "data": {
+            "default": {},
             "description": "The data/arguments of the callflow module",
+            "extended-validation": true,
```
Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/callflows.action.json]: JSON schema uses camelCase - any reason not to use it too for consistency?

Comment (lazedo) [inline applications/crossbar/priv/couchdb/schemas/callflows.action.json]: i could have added `extendedValidation` but wanted to make sure this is our extension.
feel free to change it to a better name, just make sure it doesn't collide with schema handled names  :)

Comment (lazedo) [inline applications/crossbar/priv/couchdb/schemas/callflows.action.json]: maybe `kazoo-extended-validation` or `kazooExtendedValidation` ?

Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/callflows.action.json]: `kzExtendedValidation` :) I don't know though, was more just commenting on following the camelCase convention.

**Pull request merged**